### DISCUSSION TITLE
Help on RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

### DISCUSSION THREAD

**Discussion by fudanjx** [suno-ai/bark#232] [General]
First time run my download of parameter files text_2.pt interrupted.  

2nd time run then keep givine me this error:  RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

Tried download text_2.pt manually, but not sure where to put this file.. 

please help

**Comment by gkucsko** [comment]
hmm, this is now all handled through huggingface :/
